aroma/utils.py

```diff
@@ -3,14 +3,64 @@
 import os
 import os.path as op
 import shutil
+from tempfile import mkstemp
 
 import nibabel as nib
 import numpy as np
 from nilearn import image, masking
+from scipy import ndimage
 
 LGR = logging.getLogger(__name__)
 
 
+def derive_masks(in_file, csf=None):
```

Comment:
For a start, I would always assume that a CSF TPM and a brain mask are provided.

Reply:
I would even assume all masks are provided in this first PR. Use a local MELODIC run to test everything on standard space (after resampling) to make sure that we are not introducing errors.

Reply:
To clarify, you're proposing that we add a `brain_mask` argument here, right? Instead of using `nilearn.masking.compute_epi_mask`?

I don't want to make either argument required, since we still want to support the original standard-space masks that come packaged with the code.

Reply:
Correct, and the negation of that mask happens inside the method.

The argument is better required, if you are going to calculate any mask instead of just forwarding inputs then that could be done outside of the function (one function - one responsibility)

Reply:
So then grabbing the packaged masks when native-space ones aren't available would happen in a separate function or the workflow? That seems reasonable.